### ISSUE TITLE
Hi Help me

### ISSUE THREAD

**Issue by fpemdksk** [mumble-voip/mumble#3096]
I have compiled a mumble mumblebuild based on the documentation.
 When I do nmake I get an error.

mumble_pch.hpp(57): fatal error C1083: . 'sndfile.h': No such file or directory

Can I bring sndfile.h as I am?

**Comment by davidebeatrici** [comment]
Hi,

You need libsndfile in order to compile Mumble.
http://www.mega-nerd.com/libsndfile/

**Comment by mkrautz** [comment]
@fpemdksk From the error, it looks like you're on Windows. Can you check if your MumbleBuild directory has the sndfile directory?

![image](https://cloud.githubusercontent.com/assets/36527/26236025/9478a276-3c6e-11e7-9337-11d045da9f4b.png)

**Comment by fpemdksk** [comment]
yes i have. this folder include -(FLAC), OGG, Vorbis), (lib- FLAC.lib, ogg.lib, vorbis.lib, vorbisfilelib)
However, there was no sndfile.h.
So I went into the site
I installed it differently. Added mumble / src /: sndfile.h so this error is over.
but next error 


 link /NOLOGO /DYNAMICBASE /NXCOMPAT -WX /INCREMENTAL:NO /DLL /SUBSYSTEM:
WINDOWS,6.00 /MANIFEST:embed /OUT:..\..\release\mumble_app.dll @C:\Users\Lee\App
Data\Local\Temp\nm5023.tmp
LINK : fatal error LNK1181: 'sndfile.lib' not ofen.
NMAKE : fatal error U1077: '"C:\Program Files (x86)\Microsoft Visual Studio 14.0

I am stupid and have been suffering for three months. OTL
